stats/src/lisa.rs

```diff
@@ -20,15 +33,45 @@ pub enum Quad {
     LL,
 }
 
-pub fn lisa(weights: &Weights, values: &[f64], permutations: usize, keep_sims: bool) -> LISAResult {
+/// Generates a nested set of vectors which contain permuted indices for each of the different
```

Comment:
This makes sense -- is there a good syntax for specifying n-dimensional vectors we could use to further clarify? Something like:
```
The nested vectors appears as:
[
  [ 
    [ 
      [ randomValue1,randomValue2,randomValueN] 
    ]: Permutations (eg. 999)
  ]: Count of Max Neighbors
]
```

Reply:
yeah I think  it might be worth explicitly working through an example here. It's probably the most complicated part of the code